index.html

```diff
@@ -17,59 +17,120 @@
+                    <ul class="work-skills grid-container">
+                        <li class="gird">CSS</li>
+                        <li class="gird">HTML</li>
+                        <li class="gird">Bootstrap</li>
+                        <li class="grid0">Ruby</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                <div class="work3">
+                    <img src="img/work_img/Portfolio2.png" alt="work1 image">
+                    <h3 class="work-name">Multi-Post Stories</h3>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul class="work-skills grid-container">
+                        <li class="gird">CSS</li>
+                        <li class="gird">HTML</li>
+                        <li class="gird">Bootstrap</li>
+                        <li class="grid0">Ruby</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                <div class="work4">
+                    <img src="img/work_img/Portfolio3.png" alt="work1 image">
+                    <h3 class="work-name">Multi-Post Stories</h3>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul class="work-skills grid-container">
+                        <li class="gird">CSS</li>
+                        <li class="gird">HTML</li>
+                        <li class="gird">Bootstrap</li>
+                        <li class="grid0">Ruby</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
             </div>
         </section>
-        <hr class="break-skill">
-       <section class="contact">
-           <ul class="contact-container">
-                <li>
-                    <a href="#"><img src="img/github.png" alt="github" class="github"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/linkin.png" alt="linkin" class="linkin"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/gesture.png" alt="gesture" class="gesture"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/twitter.png" alt="twitter" class="twitter"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/m.png" alt="m" class="m"></a>
-                </li>
-           </ul>
+        <section class="about-section">
```

Comment:
*Changes Required**
_Well done using the `section` html semantics. here as well_
- [ ] The section `.about-section` should have a `background image` according to the figma design. 
- [ ] This section should define css selector property `margin` defined in the design. For instance .
   ```
    .about-section {
       margin: 1rem 2rem;
     }
    ```

Reply:
the about-section already has a background image in the contact-contaniner